app/models/page_sequence.rb

```diff
@@ -2,16 +2,17 @@
 
 # Represents the pages in the claim process.
 #
-# Depending on the state of `claim`,  the sequence of slugs will change,
+# Depending on the state of `claim`, the sequence of slugs will change,
 # filtering out pages that are not relevant. For example, the sequence for a
 # claim that states it is still employed at the same school as the
-# `claim_school` will skip the `current-school` page as `current_school` is
-# inferred to be the same as `claim_school.
+# claim school will skip the `current-school` page as the current school is
+# inferred to be the same as claim school.
 class PageSequence
   SLUGS = [
     "qts-year",
+    "currently-teaching",
     "claim-school",
-    "still-teaching",
```

Comment:
Do we need to think about handling the case of a user who is on the `still-teaching` page and then refreshes after we deploy this?

Reply:
Yes. This is merging into a long-lived feature branch that we'll need to do something about rolling out because it will drastically change the flow.

Reply:
Does that mean we're planning to handle the consequences as a separate step before we release?

Reply:
Yes.